packages/cmr-client/tests/test-EarthdataLogin.js

```diff
@@ -3,7 +3,13 @@
 const test = require('ava');
```

Comment:
Please add a `retrieveEDLToken` test showing that it selects the latest token when the tokens are returned and they are not sorted by `expiration_date`.

Reply:
Correct. To make sure you're really testing it, make sure that the list you get back is not already sorted. So, for example, return three tokens and make the latest one the one in the middle of the list.